### PR TITLE
mat and toefl forms are done

### PR DESCRIPTION
**Pull request by kkaajjaall**

### CODE REVIEW

File: resources/views/extra-score-mat-form.blade.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+@extends('layouts.main')
+@section('main-content')
+<form class=" " method="post" action="{{ $actionUrl }}">
+    @csrf
+    @if (@isset($extraScoreMat))
+        {{ method_field('PUT') }}
+        <x-mat-form :extraScoreMat="extraScoreMat" />
```
Comment (ithqindia) [inline resources/views/extra-score-mat-form.blade.php:8]: `        <x-mat-form :extraScoreMat="$extraScoreMat" />
`

File: resources/views/extra-score-toefl-form.blade.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@extends('layouts.main')
+@section('main-content')
+    <form class=" " method="post" action="{{ $actionUrl }}">
+        @csrf
+        @if (@isset($extraScoreToefl))
+            {{ method_field('PUT') }}
+            <x-toefl-form :extraScoreToefl="extraScoreToefl" />
```
Comment (ithqindia) [inline resources/views/extra-score-toefl-form.blade.php:7]: `            <x-toefl-form :extraScoreToefl="$extraScoreToefl" />
`

**Review by ithqindia — CHANGES_REQUESTED** [review]

File: resources/views/extra-score-mat-form.blade.php (hunk 1 above)
Comment (kkaajjaall) [reply resources/views/extra-score-mat-form.blade.php:8]: couldn't understand the changes

**Comment by ithqindia** [comment]
Merged locally

**Pull request closed without merging**